src/main/java/core/basesyntax/dao/FileWriterImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriterMy {
+    @Override
+    public void write(String fileName,String info) {
+        try (BufferedWriter bufferedWriter = new BufferedWriter(new java.io.FileWriter(fileName))) {
+            bufferedWriter.write(info);
+        } catch (IOException e) {
+            throw new RuntimeException("Impossible to write in this file " + fileName, e);
```

Comment:
```suggestion
            throw new RuntimeException("Can’t write file with path: " + fileName, e);
```
Make the message more formal